# A service type for talker service

# Request 
string inputString

---

# Response
string outputString
 
